app/locale/ru_RU.json

```diff
@@ -0,0 +1,73 @@
+{
+    "Connecting...": "Подключение...",
+    "Disconnecting...": "Отключение...",
+    "Reconnecting...": "Перенодключение...",
+    "Internal error": "Внутренняя ошибка",
+    "Must set host": "Задайте имя сервера или IP",
+    "Connected (encrypted) to ": "Подключено (с шифрованием) к ",
+    "Connected (unencrypted) to ": "Подключено (без шифрования) к ",
+    "Something went wrong, connection is closed": "Что-то пошло не так, подключение разорвано",
+    "Failed to connect to server": "Ошибка подключения к серверу",
+    "Disconnected": "Отключено",
+    "New connection has been rejected with reason: ": "Подключиться не удалось: ",
+    "New connection has been rejected": "Подключиться не удалось",
+    "Password is required": "Требуется пароль",
+    "noVNC encountered an error:": "Ошибка noVNC: ",
+    "Hide/Show the control bar": "Скрыть/Показать контрольную панель",
+    "Move/Drag Viewport": "Переместить окно",
+    "viewport drag": "Переместить окно",
+    "Active Mouse Button": "Активировать кнопки мыши",
+    "No mousebutton": "Отключить кнопки мыши",
+    "Left mousebutton": "Левая кнопка мыши",
+    "Middle mousebutton": "Средняя  кнопка мыши",
+    "Right mousebutton": "Правая кнопка мыши",
+    "Keyboard": "Клавиатура",
+    "Show Keyboard": "Показать клавиатуру",
+    "Extra keys": "Доп. кнопки",
+    "Show Extra Keys": "Показать дополнительные кнопки",
+    "Ctrl": "Ctrl",
+    "Toggle Ctrl": "Передать нажатие Ctrl",
+    "Alt": "Alt",
+    "Toggle Alt": "Передать нажатие Alt",
+    "Toggle Windows": "Переключение вкладок",
+    "Windows": "Вкладка",
+    "Send Tab": "Передать нажатие Tab",
+    "Tab": "Tab",
+    "Esc": "Esc",
+    "Send Escape": "Передать нажатие Escape",
+    "Ctrl+Alt+Del": "Strg+Alt+Entf",
+    "Send Ctrl-Alt-Del": "Передать нажатие Ctrl-Alt-Del",
+    "Shutdown/Reboot": "Выключить/Перезагрузить",
+    "Shutdown/Reboot...": "Выключить/Перезагрузить...",
+    "Power": "Питание",
+    "Shutdown": "Выключить",
+    "Reboot": "Перезагрузить",
+    "Reset": "Сброс",
+    "Clipboard": "Буфер обмена",
+    "Clear": "Очистить",
+    "Fullscreen": "Во весь экран",
+    "Settings": "Настройки",
+    "Shared Mode": "Общий режим",
+    "View Only": "Просмотр",
+    "Clip to Window": "В окно",
+    "Scaling Mode:": "Масштаб:",
+    "None": "Нет",
+    "Local Scaling": "Локльный масштаб",
+    "Remote Resizing": "Удаленный масштаб",
+    "Advanced": "Дополнительно",
+    "Repeater ID:": "Идентификатор ID:",
+    "WebSocket": "WebSocket",
+    "Encrypt": "Шифрование",
+    "Host:": "Сервер:",
+    "Port:": "Порт:",
+    "Path:": "Путь:",
+    "Automatic Reconnect": "Автоматическое переподключение",
+    "Reconnect Delay (ms):": "Задержка переподключения (мс):",
+    "Show Dot when No Cursor": "Показать точку вместо курсора",
+    "Logging:": "Лог:",
+    "Disconnect": "Отключение",
+    "Connect": "Подключение",
+    "Password:": "Пароль:",
+    "Send Password": "Пароль: ",
+    "Cancel": "Выход"
+}
```

Comment:
You can skip this commit, updating the json files will be done as part of the release procedure.

Reply:
Miss at merge.